swipeable-rv/src/main/java/io/huannguyen/swipeablerv/SWItemRemovalListener.java

```diff
@@ -35,6 +39,8 @@
      *         The item being added back
      * @param position
      *         The position of the item being added back
+     * @param direction
```

Comment:
Hey @huan-nguyen this was a miss on my part. The direction info would be needed for an undo to be able to know which action was taken and thus how to undo that action.

I've added this parameter & associated data in and re-submitted  👍🏻